tools/data_builder.py

```diff
@@ -177,7 +181,8 @@ def to_json_file(data, target_path):
     data_with_meta = {
         "edit_at": "https://github.com/2020PB/police-brutality",
         "help": "ask @ubershmekel on twitter",
-        "data": data,
+        "timestamp": str(datetime.now()),

```

Comment:
Please use a more standard time format such as:

```
>>> from datetime import datetime, timezone
>>> datetime.now(timezone.utc).isoformat()
'2015-01-27T05:57:31.399861+00:00'
```
Also I would use the key `updated_at` as it's less ambiguous.

Reply:
Also I would want all the files to use the same timestamp